### PR TITLE
To support a test case about disable dns and dhcp

### PR DESCRIPTION
**Pull request by yalzhang**
Network can be set as dns disabled or dhcp disabled. Set
<dns enable='no'/> can disable the dns function of the vm. And
delete the dhcp range can disable dhcp for the network.

Signed-off-by: Yalan <yalzhang@redhat.com>

### CODE REVIEW

**Comment by yalzhang** [comment]
test result refer to https://github.com/autotest/tp-libvirt/pull/3674#issuecomment-889751713

File: virttest/libvirt_xml/network_xml.py [hunk 1]
```diff
@@ -173,13 +173,15 @@ class DNSXML(base.LibvirtXMLBase):
             List of host name
     """
 
-    __slots__ = ('dns_forward', 'txt', 'forwarders', 'srv',
+    __slots__ = ('enable', 'dns_forward', 'txt', 'forwarders', 'srv',
```
Comment (chunfuwen) [inline virttest/libvirt_xml/network_xml.py:176]: add one line explanation above for newly added attribute name enable

Comment (yalzhang) [reply virttest/libvirt_xml/network_xml.py:181]: Updated, Thank you!

**Review by chunfuwen — APPROVED** [review]
LGTM

File: virttest/libvirt_xml/network_xml.py [hunk 2]
```diff
@@ -159,27 +159,34 @@ def __init__(self, virsh_instance=base.virsh):
 class DNSXML(base.LibvirtXMLBase):
 
     """
-    IP address block, optionally containing DHCP range information
+    DNS block, contains configuration information for the network's DNS server
 
     Properties:
+        enable:
+            String, 'yes or 'no'. 'Set to "no", then no DNS server
```
Comment (kylazhang) [inline virttest/libvirt_xml/network_xml.py:166]: please update the (**'**) place

**Review by kylazhang — COMMENTED** [review]
Others are ok for me

**Comment by kylazhang** [comment]
Also please rebase and submit again to avoid trivals ci check failure

File: virttest/libvirt_xml/network_xml.py (hunk 2 above)
Comment (yalzhang) [reply virttest/libvirt_xml/network_xml.py]: updated, Thank you!

**Review by kylazhang — APPROVED** [review]

**Pull request merged**